docs/trex_typescript.md

```diff
@@ -78,37 +73,31 @@ You can set the `tsc` compiler options in the `tsconfig.json` file for the proje
     "typeRoots": ["./node_modules/@tableau", "./node_modules/@types"]
   }
 }
-
-
-
 ```
 
 ## Compile your code with the TypeScript compiler
 
 If you need to install the compiler, see [TypeScript in Visual Studio Code](https://code.visualstudio.com/docs/languages/typescript?=target="_blank"). Visual Studio Code supports TypeScript, but does not automatically include the TypeScript compiler (`tsc`). The TypeScript compiler transpiles your TypeScript source code to JavaScript. You include the JavaScript output in your extension `.html` file(s).
 
-
 ## Use the compiled JavaScript output in your extension
 
-The TypeScript compiler (`tsc`) transpiles the source code into JavaScript. You just need to include the path to the JavaScript file in your extension code.  
+The TypeScript compiler (`tsc`) transpiles the source code into JavaScript. You just need to include the path to the JavaScript file in your extension code.
 
-For example, the extensions in the Samples-Typescript folder all link to Extensions API JavaScript library `tableau.extensions.1.latest.js` and to the compiled JavaScript file for the extension (not the TypeScript source file). 
+For example, the extensions in the Samples-Typescript folder all link to Extensions API JavaScript library `tableau.extensions.1.latest.js` and to the compiled JavaScript file for the extension (not the TypeScript source file).
 In the HTML code for the extensions, the JavaScript files are referenced. The following example, links to the `datasources.js` file.
 
 ```html
-    <!-- Extensions Library (this will be hosted on a CDN eventually) -->
-    <script src="../../lib/tableau.extensions.1.latest.js"></script>
-
-    <!-- The  extension code (webpack) -->
-    <script src="../../dist/datasources.js"></script>
-
+<!-- Extensions Library (this will be hosted on a CDN eventually) -->
```

Comment:
This is hosted on our sandboxing server if we want to start giving that url out to customers

Reply:
Should we put...
<!-- Extensions Library (also hosted at ...) -->
How do I find the sandboxing server url?

Reply:
Markdown removed most of my comment. I'm going to leave it as is this time around.